core/src/test/java/org/apache/calcite/avatica/UnregisteredDriverTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.avatica;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.sql.Driver;
+import java.sql.SQLException;
+
+/**
+ * Test class for {@link UnregisteredDriver}.
+ */
+public class UnregisteredDriverTest {
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+
+  @Test public void testNullUrl() throws SQLException {
```

Comment:
Just to be sure that things will not break in the future add also an extra unit test for `driver.connect`

Reply:
Sorry, I did not get your point by `Just to be sure that things will not break in the future`